pkg/transformer/kubernetes/k8sutils.go

```diff
@@ -318,6 +319,13 @@ func (k *Kubernetes) UpdateKubernetesObjects(name string, service kobject.Servic
 		template.Spec.Containers[0].TTY = service.Tty
 		template.Spec.Volumes = volumes
 
+		memoryResourceList := api.ResourceList{
+			api.ResourceMemory: *resource.NewQuantity(
+				int64(service.MemLimit), "RandomStringForFormat")}
+
+		template.Spec.Containers[0].Resources.Limits = memoryResourceList
+		template.Spec.Containers[0].Resources.Requests = memoryResourceList
```

Comment:
@surajssd @kadel thanks, will look into this. Let's continue the review once I remove the WIP.